netclient.rst

```diff
@@ -204,6 +204,52 @@ By using this method, you can run many netclients on the same host and just incr
 
 
 
+******************
+Firewall Settings
```

Comment:
Firewall Settings should be moved to the prereqs section (this is added in the develop branch, pull the latest changes to your branch). we should include firewall settings required for the netmaker server as well
we have an old section for the server already, refer here - https://docs.netmaker.io/manual-install.html#open-firewall